scripts/init.sh

```diff
@@ -2,7 +2,16 @@
 
 set -e
 
+echo "Setup git hooks"
+git config core.hooksPath hooks/
```

Comment:
I like to be explicit here:
```suggestion
git config --local core.hooksPath hooks/
```